src/crypto/crypto.js

```diff
@@ -468,12 +468,26 @@ export function generateSessionKey(algo) {
 /**
  * Get implementation of the given AEAD mode
  * @param {enums.aead} algo
+ * @param {Boolean} [rejectExperimentalGCM] - whether to throw on non-standard, legacy `experimentalGCM` algo
  * @returns {Object}
  * @throws {Error} on invalid algo
  */
-export function getAEADMode(algo) {
-  const algoName = enums.read(enums.aead, algo);
-  return mode[algoName];
+export function getAEADMode(algo, rejectExperimentalGCM = false) {
```

Comment:
Bit nitpicky, but probably it's safer to swap the meaning and the default:
```suggestion
export function getAEADMode(algo, allowExperimentalGCM = false) {
```
That way, it'll do the right thing by default if we use this function in new places (like persistent symmetric keys).